### PR TITLE
Remove pending summary logging 

### PR DESCRIPTION
**Pull request by NicholasCouri**
Remove pending summary logging and make sure we log undefined in case the summarizer node has been disposed of.

### CODE REVIEW

**Review by wes-carlson — APPROVED** [review]

File: packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts [hunk 1]
```diff
@@ -227,15 +227,12 @@ export class SummarizerNode implements IRootSummarizerNode {
 
             const props = {
                 summaryRefSeq,
-                pendingSize: this.pendingSummaries.size,
-                pendingHandle: this.pendingSummaries.size > 0 ? this.pendingSummaries.keys[0] : undefined,
-                pendingSeqNumber: this.pendingSummaries.size > 0 ?
-                    this.pendingSummaries.values[0].referenceSequenceNumber : undefined,
```
Comment (wes-carlson) [inline packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts:233]: This mostly looks fine to me, except `this.pendingSummaries.values[0].referenceSequenceNumber` which looks wrong, but could probably just be changed to ` this.pendingSummaries.values[0]?.referenceSequenceNumber`

Comment (NicholasCouri) [reply packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts:233]: yes, I believe we should be fine. From all the loggings, there wasn't a single entry in which the size > 0 - so I don't think we need the other fields.

File: packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts [hunk 2]
```diff
@@ -227,15 +227,12 @@ export class SummarizerNode implements IRootSummarizerNode {
...
+                pendingSize: this.pendingSummaries.size ?? undefined,
             };
             this.defaultLogger.sendTelemetryEvent({
                 eventName: "PendingSummaryNotFound",
                 proposalHandle,
-                referenceSequenceNumber: this.referenceSequenceNumber,
+                referenceSequenceNumber: this.referenceSequenceNumber ?? undefined,
```
Comment (curtisman) [inline packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts:235]: why is this necessary? this.referenceSequenceNumber is always a number, no?  This would turn some number (like 0) to undefined?

Comment (NicholasCouri) [reply packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts:235]: Indeed, it is not required - the error happened while trying to get the pending summary value.
